### PR TITLE
Add edit/update TodoList functionality

### PR DESCRIPTION
**Pull request by lenayee7**
# Overview
Allows you to Update a TodoList

## Testing
- [ ] Go to TodoList show page
- [ ] Click Edit
- [ ] Update your TodoList
- [ ] Click Submit
- [ ] Verify Flash Notice and TodoList is updated

### CODE REVIEW

File: app/controllers/todo_lists_controller.rb [hunk 1]
```diff
@@ -13,16 +13,27 @@ def new
 		@todo_list = TodoList.new
 	end
 
+	def edit
+		@todo_list = TodoList.find(params[:id])
```
Comment (edmundonm) [inline app/controllers/todo_lists_controller.rb]: The indentation is too much, let's move to using 2 spaces for each tab.

**Review by edmundonm — CHANGES_REQUESTED** [review]
fix indentation

File: app/controllers/todo_lists_controller.rb (hunk 1 above)
Comment (lenayee7) [reply app/controllers/todo_lists_controller.rb]: I tried hard and we did it!
The TodoLists Controller only. 😉

**Review by edmundonm — APPROVED** [review]
well done  👍

**Pull request merged**